applications/GeoMechanicsApplication/tests/cpp_tests/custom_utilities/test_constitutive_law_utilities.cpp

```diff
@@ -130,4 +131,51 @@ KRATOS_TEST_CASE_IN_SUITE(RaiseADebugErrorWhenIndexInUMatParametersIsOutOfBounds
         "Got out-of-bounds INDEX_OF_UMAT_C_PARAMETER (material ID: 0): 3 is not in range [1, 2]");
 }
 
+KRATOS_TEST_CASE_IN_SUITE(ConstitutiveLawUtilities_GetStateVariableIndex, KratosGeoMechanicsFastSuiteWithoutKernel)
+{
+    // Arrange
+    auto                     properties       = Properties{};
+    ConstitutiveLaw::Pointer constitutive_law = Kratos::make_shared<MockConstitutiveLaw>();
+    properties.GetValue(CONSTITUTIVE_LAW)     = constitutive_law;
+
+    // Act and Assert
+    constexpr auto expected_index = 1;
+    KRATOS_EXPECT_EQ(ConstitutiveLawUtilities::GetStateVariableIndex(STATE_VARIABLE_2), expected_index);
+}
+
+KRATOS_TEST_CASE_IN_SUITE(ConstitutiveLawUtilities_CheckStrainSize, KratosGeoMechanicsFastSuiteWithoutKernel)
+{
+    // Arrange
+    auto                     properties       = Properties{};
+    ConstitutiveLaw::Pointer constitutive_law = Kratos::make_shared<MockConstitutiveLaw>();
+    properties.GetValue(CONSTITUTIVE_LAW)     = constitutive_law;
```

Comment:
Nitpicking, but this could be a minor simplification:
```suggestion
    properties.GetValue(CONSTITUTIVE_LAW)     = Kratos::make_shared<MockConstitutiveLaw>();
```

Reply:
joint